docs/Concepts/SecurityFramework.md

```diff
@@ -0,0 +1,135 @@
+---
+description: Security framework overview
+---
+
+# Security framework
+
+This topic describes high-level security best practices for:
+
+- Creating a GoQuorum [consortium](#consortium).
+- Securing the components of a [GoQuorum node](#goquorum-node).
+- [Monitoring](#monitoring) a GoQuorum network.
+- Using the Tessera [transaction manager](#transaction-manager).
+
+You can view a [list of additional network and dapp security resources](#resources).
+
+## Consortium
+
+A consortium network connects multiple independent networks.
+Consortiums come with risks and liabilities that must be managed.
```

Comment:
This sentence makes me question "what risks and liabilities?"  Maybe adding an example to clarify. Something like:

```suggestion
Because of the nature of shared ownership, Consortiums come with risks like accidental data exposure and potential liabilities that must be managed.
```